plugins/org.jkiss.dbeaver.erd.model/src/org/jkiss/dbeaver/erd/model/ERDAssociation.java

```diff
@@ -68,25 +68,24 @@ public ERDAssociation(ERDElement<?> sourceEntity, ERDElement<?> targetEntity, bo
 
     /**
      * Constructor for physical association
-     * @param association physical FK
+     *
+     * @param association  physical FK
      * @param sourceEntity fk table
      * @param targetEntity pk table
-     * @param reflect reflect flag
+     * @param reflect      reflect flag
      */
-	public ERDAssociation(DBSEntityAssociation association, ERDEntity sourceEntity, ERDEntity targetEntity, boolean reflect)
-	{
-		super(association);
-		this.targetEntity = targetEntity;
-		this.sourceEntity = sourceEntity;
-
-		// Resolve association attributes
+    public ERDAssociation(DBSEntityAssociation association, @NotNull ERDEntity sourceEntity, @NotNull ERDEntity targetEntity,
```

Comment:
Code style: all parameters on the same line or each parameter on its own.
Also we use `\t) {\n` as header delimiter.

Reply:
Fixed